convert2rhel/systeminfo.py

```diff
@@ -350,14 +350,19 @@ def _check_internet_access(self):
         try:
             response = urllib.request.urlopen(CHECK_INTERNET_CONNECTION_ADDRESS)
             response.close()
-            self.logger.info("Internet connection available.")
+            self.logger.info(
+                "Successfully connected to address '%s', internet connection seems to be available."
+                % CHECK_INTERNET_CONNECTION_ADDRESS
+            )
             return True
         except urllib.error.URLError as err:
-            self.logger.debug("Faild to retrieve data from host, reason: %s", err.reason)
             self.logger.warning(
-                "Couldn't connect to the address '%s', assuming no internet connection is present.",
+                "There was a problem while trying to connect to '%s' to check internet connectivity, "
+                "it seems that the host appears to be offline or not responding. Convert2RHEL will continue the conversion "
+                "but without support for online resources.",
```

Comment:
Continuing on [the last convo](https://github.com/oamg/convert2rhel/pull/587#discussion_r986900426). Maybe something like this. `Inaccessible` here can refer to both that it's blocked by the customer or that the server is down.

@bocekm what do you think?

> There was a problem while trying to connect to '%s' to check internet connectivity, it appears that the host is inaccessible. This could be due to the host being offline, or the network blocking access to the endpoint. Some checks and actions will be skipped, which will decrease the chance for a successful conversion.

Reply:
I feel that the part "it appears that the host is inaccessible" is not necessary there, it adds no additional information.
Similarly, the part